HW1/P5/P5_bfs.py

```diff
@@ -0,0 +1,73 @@
+#NOTE: File P5.py should be used to call this function below to find
+#shortest path between two nodes
+
+import numpy as np
+
+def copartitioned(RDD1, RDD2):
+    "check if two RDDs are copartitioned"
+    return RDD1.partitioner == RDD2.partitioner
+    
+sequence = []
+  
+def bfs(SOURCE,Graph,sc,DEST=False):
+    #set source as source node and distance to 0    
+    node1 = [(SOURCE,0)]
+    node1 = sc.parallelize(node1).partitionBy(128)
```

Comment:
Why 128? I seem to remember 256 being the way to go? Probably doesn't matter all that much, though.